src/cursor/cur_std.c

```diff
@@ -582,6 +595,100 @@ err:	API_END(session, ret);
 }
 
 /*
+ * __cursor_modify --
+ *	WT_CURSOR->modify default implementation.
+ */
+static int
+__cursor_modify(WT_CURSOR *cursor, WT_MODIFY *entries, int nentries)
```

Comment:
I'd rather ``nentries`` was a signed size-defined type, maybe ``uint32_t``. Is there a reason to make it an ``int``?